### ISSUE TITLE
Translate everything to english

### ISSUE THREAD

**Issue by marco44** [rjuju/postgres-manage#6]
Maybe it would be worth it to translate comments to english, and start internationalizing the few messages we are displaying (or put them in plain english ?)

**Comment by rjuju** [comment]
+1 for translating everything in english.  I'm not sure that adding translation support is worth it, but it's up to you.

Please keep backward compatibility for parameters such as "parallelisme" :)

**Comment by marco44** [comment]
Forgot to close :)

**Comment by marco44** [comment]
Wooops, the patch from superette is still missing...
